src/DelayId.cc

```diff
@@ -155,14 +138,13 @@ DelayId::bytesIn(int qty)
 
     assert ((unsigned short)(pool() - 1) != 0xFFFF);
```

Comment:
If we are going to simplify this code in this PR, then I would assert condition B that this code actually uses (rather than a condition A that implies B):

```suggestion
    assert(compositeId);
```

Reply:
I think we can simply drop this assertion, because we already check the condition at the function beginning.  I wonder why they use this cumbersome expression instead of a simpler pool() check.